UI/Sources/ShoppingCart/Views/CartWeightView.swift

```diff
@@ -38,7 +39,7 @@ struct CartWeightView: View {
     @ViewBuilder
     var minusImage: some View {
         Image(systemName: "trash")
-            .foregroundColor(.accentColor)
+            .foregroundColor(.projectPrimary())
```

Comment:
Don't we need the @Environment(.project) here?

Reply:
42149bf95c5bcc1a82518cc5ba35d7ae5fe67014